ldm/models/diffusion/ksampler.py

```diff
@@ -53,17 +53,9 @@ def make_schedule(
             ddim_eta=0.0,
             verbose=False,
         )
-        self.model = outer_model
+        self.model          = outer_model
```

Comment:
Please stop doing this, *if only* because of https://www.reddit.com/r/webdev/comments/4trfxm/comment/d5jps8k

The research (yes, there is actual research on readability - in a former life I was a graphic design student (and before that human factors/ergonomics)) shows that column aligning things like this, **unless they are in an actual table**, actually harms readability.

It is **harder** for the eye to track - literally nothing - across what ends up being a randomly-sized gap, than it is for it to skip over a consistent and uniform "\<space>\<operator>\<space>"

Reply:
Huh. I find it much easier on the eyes.